bin/zks

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
 cd "$ZK_PATH"
 
-fzf --ansi --height 100% --preview 'zk-fts-search --backup -f {} {q} | bat --language md --style=plain --color always' \
+# mini tl;dr for binds
+# enter   => select entry, and left tmux window do nvim stuff
+# ctrl-o  => insert the file "hovered" as a wikilink/freelink style eg. [[file]] (expects left nvim)
+# alt-s   => some search for similar notes (TF-IDF) optional, see https://github.com/sirupsen/zk/pull/1
+#
+# changes to fzf input will reload but with a small timeout
+# it was opening the sqlite db too quickly (could be mac specific), and would get "db locked" errors
+
+rg --files --no-hidden --max-depth 1 --glob '*.md' |\
```

Comment:
I think this was pretty useful, or at least it felt more "right" to me, since the scripts also do this filtering internally it only makes sense to filter it out for `fzf` too.